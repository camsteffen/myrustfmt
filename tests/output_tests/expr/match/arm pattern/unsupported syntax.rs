// test-kind: no-change

fn test() {
    match x {
        box a => a,
    }
}
