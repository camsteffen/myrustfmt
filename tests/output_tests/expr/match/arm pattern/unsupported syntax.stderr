Unsupported syntax at 3:9
